source/exe/BUILD

```diff
@@ -52,6 +52,7 @@ envoy_cc_library(
     deps = [
         ":envoy_common_lib",
         ":hot_restart_lib",
+        ":sigaction_lib",
```

Comment:
Can you use a `select` statement here and condition on `//bazel:disable_signal_trace` to avoid the link dependency on backward.hpp when we do the google import?